code/__DEFINES/traits.dm

```diff
@@ -630,6 +630,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_INTROVERT "introvert"
 #define TRAIT_ANXIOUS "anxious"
 #define TRAIT_SMOKER "smoker"
+#define TRAIT_POSTERBOY "poster_boy"
+#define TRAIT_THROWINGARM "throwingarm"
```

Comment:
```suggestion
#define TRAIT_THROWINGARM "throwing_arm"
```

snake case is swag

Reply:
this ones funny because you see me do it the correct way for the other quirk